recipes/feelnc/meta.yaml

```diff
@@ -15,7 +15,6 @@ build:
 
 requirements:
   build:
-    - perl ==5.22.0.1
```

Comment:
perl should be given if perl is needed. This is a special case afaik for all scripting languages. Some for R or python.

Reply:
ok, good to know, I wasn't 100% sure